src/backend/tools/slack/tool.py

```diff
@@ -0,0 +1,55 @@
+from typing import Any, Dict, List
+
+from backend.config.settings import Settings
+from backend.crud import tool_auth as tool_auth_crud
+from backend.services.logger.utils import LoggerFactory
+from backend.tools.base import BaseTool
+from backend.tools.slack.constants import SEARCH_LIMIT, SLACK_TOOL_ID
+from backend.tools.slack.utils import get_slack_service
+
+logger = LoggerFactory().get_logger()
+
+
+class SlackTool(BaseTool):
+    """
```

Comment:
Can you add all the object types it searches (eg Files/Messages)

Reply:
Ty, done